lib/matplotlib/backend_bases.py

```diff
@@ -2674,6 +2674,15 @@ def __init__(self, canvas, num):
                 'key_press_event',
                 self.key_press)
 
+        self.toolmanager = None
+        self.toolbar = None
+
+        @self.canvas.figure.add_axobserver
+        def notify_axes_change(fig):
```

Comment:
This also removes the method from all of the sub-classes.

Reply:
What method?

Reply:
:sheep: never mind, I thought this was a top-level method.